diff --git a/util/install_helper.sh b/util/install_helper.sh
index cb649a7..e3acade 100755
--- a/util/install_helper.sh
+++ b/util/install_helper.sh
@@ -39,8 +39,6 @@ fi
 install -D -m 644 "${MESON_SOURCE_ROOT}/util/udev.rules" \
         "${DESTDIR}${udevrulesdir}/99-fuse3.rules"
 
-install -D -m 755 "${MESON_SOURCE_ROOT}/util/init_script" \
-        "${DESTDIR}/etc/init.d/fuse3"
 
 
 if test -x /usr/sbin/update-rc.d && test -z "${DESTDIR}"; then
